src/pages/home/report/ReportActionsView.js

```diff
@@ -199,7 +200,7 @@ function ReportActionsView(props) {
     const loadNewerChats = useMemo(
         () =>
             _.throttle(({distanceFromStart}) => {
-                if (props.isLoadingNewerReportActions || props.isLoadingInitialReportActions) {
+                if (props.isLoadingNewerReportActions || props.isLoadingInitialReportActions || isNewestReportAction) {
```

Comment:
It seems like this `isNewestReportAction` will most likely always be `true`. Since as soon as we have the actions for display, we are setting the first report action to have `isNewestReportAction: true` and checking the same here.

When can this value be false, so that we will `loadNewerChats` function will fetch new data?

I am not sure if I had understood clearly, kindly clarify if I misunderstood.

@perunt

Reply:
Before all, we must remember that we cannot show some specific range of reportActions in the list. We constantly mix them all and sort them. Comment linking brings the ability to cut sequences chank of sorted messages and will help not to include missed places.
Firstly, we need to figure out when we need to fetch new report actions. Answer - when we locate not on the end of the list. What I mean:
- when you were offline and received 1000 report actions. Once you return online, call `OpenReport` and get the latest 50 reportAction. So your last message, the latest before moving offline now, is different. 
You have `current report action -> gap with 950 items -> the latest 50 elements.` In this case, we need to fetch new items when we reach that gap with 950 el
- the same stuff for comment linking. Once we were navigated to the middle of a chat, we should be able to fetch a newer report action when scrolling down

In the rest cases, we do not need this functionality since Pusher keeps this up-to-date.

Reply:
Discussed with @perunt Offline and it seems for now `isNewestReportAction` will be rarely false. As part next sequent PRs `loadNewerChats` will be further updated to fullest extent.